runtime/cudaq/platform/mqpu/nvqc.config

```diff
@@ -27,19 +27,19 @@ PLATFORM_EXTRA_ARGS=""
 #   --remote-sim-backend foo
 while [ $# -gt 1 ]; do
 	case "$1" in
-	--nvcf-backend)
+	--nvqc-backend)
 		PLATFORM_EXTRA_ARGS="$PLATFORM_EXTRA_ARGS;backend;$2"
 		;;
-	--nvcf-nqpus)
+	--nvqc-nqpus)
 		PLATFORM_EXTRA_ARGS="$PLATFORM_EXTRA_ARGS;nqpus;$2"
 		;;
-	--nvcf-function-id)
+	--nvqc-function-id)
```

Comment:
Do you expect to create an additional setting for nr gpus per virtual processor or did you intend the function-id parameter to meet this need?

Reply:
Fixed in https://github.com/NVIDIA/cuda-quantum/commit/ae21a9b4165762420178cb7cbfa3a4cdf89892af.
Number of GPUs is now a supported option.